test/fixtures/node.yml

```diff
@@ -6,7 +6,6 @@ one:
   status: 1
   type: "note"
   uid: 2
-  nid: 1
```

Comment:
Great! I'll give these a try today. Thanks!
On Mar 25, 2016 6:24 AM, "Ananya Maiti" <notifications@github.com> wrote:

> In test/fixtures/node.yml
> <https://github.com/publiclab/plots2/pull/452#discussion_r57433766>:
>
> > @@ -6,7 +6,6 @@ one:
> >    status: 1
> >    type: "note"
> >    uid: 2
> > -  nid: 1
>
> Giving explicit value to a primary key doesn't allow to associate it with
> other model. Auto generating it allow association with other model.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/publiclab/plots2/pull/452/files/bc98bcdb024f1f6e1a171252a0e6bde9690ed38b..2d06840febde93431470db0fa6dd51cb80c87aed#r57433766>
>